main.go

```diff
@@ -26,6 +27,11 @@ import (
 	"github.com/bitrise-tools/go-steputils/tools"
 )
 
+const (
+	maxTimeout = 1800
+	timeoutPattern = `^\d+$|^\d+\.\d{1,9}$` // see: https://firebase.google.com/docs/test-lab/reference/testing/rest/v1/projects.testMatrices#testspecification
```

Comment:
As I read the specs, I found this:
A duration in seconds with up to nine fractional digits, terminated by 's'. Example: "3.5s".
I am missing the s from the end of the regex.

Reply:
Very good catch! However, the `s` is, in fact, appended to the value at a later point.

https://github.com/bitrise-steplib/steps-virtual-device-testing-for-ios/blob/master/main.go#L219